packages/components/addon/components/hds/dropdown/index.hbs

```diff
@@ -12,18 +12,22 @@
     }}
   </:toggle>
   <:content as |c|>
-    <ul class={{this.listClassNames}} {{style width=@width}}>
-      {{yield
-        (hash
-          close=c.close
-          CopyItem=(component "hds/dropdown/list-item/copy-item")
-          Description=(component "hds/dropdown/list-item/description")
-          Generic=(component "hds/dropdown/list-item/generic")
-          Interactive=(component "hds/dropdown/list-item/interactive")
-          Separator=(component "hds/dropdown/list-item/separator")
-          Title=(component "hds/dropdown/list-item/title")
-        )
-      }}
-    </ul>
+    <div class={{this.contentClassNames}} {{style width=@width height=@height}}>
+      {{yield (hash Header=(component "hds/dropdown/header"))}}
+      <ul class="hds-dropdown__list" role="listbox">
+        {{yield
+          (hash
+            close=c.close
+            CopyItem=(component "hds/dropdown/list-item/copy-item")
+            Description=(component "hds/dropdown/list-item/description")
+            Generic=(component "hds/dropdown/list-item/generic")
+            Interactive=(component "hds/dropdown/list-item/interactive")
+            Separator=(component "hds/dropdown/list-item/separator")
+            Title=(component "hds/dropdown/list-item/title")
+          )
+        }}
+      </ul>
+      {{yield (hash close=c.close Footer=(component "hds/dropdown/footer"))}}
```

Comment:
Gave it another go now. Unfortunately, if `close` is not forwarded via hashing it can't be called by elements in the footer as in the example below.

```hbs
<Hds::Dropdown @listPosition="left" as |dd|>
  <dd.ToggleButton @text="Checkbox" @color="secondary" />
  <dd.Footer>
    <Hds::Button @text="Apply filters" @isFullWidth={{true}} {{on "click" dd.close}} @size="small" />
  </dd.Footer>
</Hds::Dropdown>
```